src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core'
+import { IconButton, IconButtonSize } from '../../enums/icon-button'
+
+export interface MultiSelectOptions {
+  label: string
+  id: string
+  classesOnSelected?: string
+  classesOnList?: string
+}
+
+@Component({
+  selector: 'sofie-multi-select',
+  templateUrl: './multi-select.component.html',
+  styleUrls: ['./multi-select.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MultiSelectComponent {
+  @Input() public options: MultiSelectOptions[]
+  @Input() public selectedOptionsIds: string[]
+  @Input() public placeholder: string
+  @Input() public showOnlyItemsCountLabel?: string
+
+  @Output() public readonly onItemSelectionChange: EventEmitter<string[]> = new EventEmitter<string[]>()
+
+  public readonly IconButton = IconButton
+  public readonly IconButtonSize = IconButtonSize
+
+  public getOptionById(optionId: string): MultiSelectOptions | undefined {
```

Comment:
This method is only required because you chose to let the responsibility of keeping track of the selected options _outside_ of the MultiSelect component _and_ because you are only giving the MultiSelect components the ids of the select options instead of giving them the entire option. We are writing object-oriented programming and we should utilize it.